src/test/kotlin/com/exactpro/th2/common/event/TestEvent.kt

```diff
@@ -52,6 +52,7 @@ class TestEvent {
     private val data = EventUtils.createMessageBean("0123456789".repeat(20))
     private val dataSize = MAPPER.writeValueAsBytes(listOf(data)).size
     private val bigData = EventUtils.createMessageBean("0123456789".repeat(30))
+    private val rawBody = "Test raw data longer than 25 characters"
```

Comment:
question: What is it about 25 characters? If it has any meaning this should be explained. Otherwise, I believe, we should not leave text that causes questions when you look at it

Reply:
Added description for rawBody variable